### PR TITLE
Fix m420 with mbl

### PR DESCRIPTION
**Pull request by InsanityAutomation**
With mesh bed leveling, manual probing is implied and functions need to be included.

### CODE REVIEW

File: Marlin/src/module/probe.h [hunk 1]
```diff
@@ -67,7 +67,7 @@
 
 #endif
 
-#if HAS_BED_PROBE || ENABLED(PROBE_MANUALLY)
+#if HAS_BED_PROBE || HAS_LEVELING
```
Comment (tpruvot) [inline Marlin/src/module/probe.h:70]: Marlin/src/inc/Conditionals_post.h:1424:
`#define HAS_LEVELING    (HAS_ABL_OR_UBL || ENABLED(MESH_BED_LEVELING))`

HAS_LEVELING should be enough no ?

Comment (tpruvot) [reply Marlin/src/module/probe.h:70]: hmm, unsure, maybe not.... :p

Comment (thinkyhead) [reply Marlin/src/module/probe.h:70]: This will work for 90% of cases, if not all.

**Comment by afaulkner420** [comment]
Will this solve this issue? 
https://github.com/MarlinFirmware/Marlin/issues/16458

**Comment by InsanityAutomation** [comment]
@afaulkner420 no, this is a compile time issue not runtime.

**Comment by afaulkner420** [comment]
@InsanityAutomation ah ok. My bad. Is the lack of MBL with dual extruders a known issue do you know? Sorry for the hijack!

**Pull request merged**